### ISSUE TITLE
Request: Freedom parrot and others from Reddit

### ISSUE THREAD

**Issue by jbcpollak** [jmhobbs/cultofthepartyparrot.com#133]
Sorry if this has already been discussed, but I'd love to have Freedom Parrot for Slack, and a few of the others from here:

https://www.reddit.com/r/PartyParrot/wiki/howto

I looked into getting them, but it seems they are all part of one giant png with sprites, and I'm not sure how to slice that up.

**Comment by jmhobbs** [comment]
Yeah, looks like all the parrots are a giant sprite sheet animated by CSS.  Strange.

**Comment by melinerunen** [comment]
http://www.charlesstover.com/spritesheet2gif
I made a quick test and it works wonders:
<img width="1127" alt="image" src="https://user-images.githubusercontent.com/1240420/30927114-bd956122-a38d-11e7-881a-6fff3db1d6e1.png">

What you have to do is to cut one column of the png without borders and in the webpage set it to automatically and take in account the pixels between sprites: 
<img width="74" alt="image" src="https://user-images.githubusercontent.com/1240420/30927182-0d65221e-a38e-11e7-9a9d-67f7876ca96e.png">

The png is the following:

https://b.thumbs.redditmedia.com/qt8PYHHuusGxTvtgu6pSC-LkGI89--PWNHUjGP9OyCc.png

When you get your gif properly made, then you can play with the framerae and transparency(in the matte section)

Enjoy it :ultrafastparrot:
